host-interaction/file-system/reference-absolute-stream-path-windows.yml

```
@@ -0,0 +1,28 @@
+rule:
+  meta:
+    name: reference absolute stream path Windows
+    namespace: host-interaction/file-system
+    authors:
+      - blas.kojusner@mandiant.com
+    scope: basic block
```

```suggestion
    scope: function
```

im not quite sure if this should be file, function, or basic block. i dont think we have guidance. im inclined to suggest "function" since analysts operate at this level more often.

thoughts @mr-tz or @mike-hunhoff ?

---

I think original guidance was to keep it to the tighter scope, but BB or function here are fine with me.

---

keeping this as BB works for me, thanks @mr-tz